xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -184,7 +184,15 @@ The property returns a string that indicates the name of the currently executing
       <Docs>
         <summary>Gets the platform architecture on which the current app is running.</summary>
         <value>The platform architecture on which the current app is running.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
the text above doesn't tell me whether I should expect the underlying architecture, or the emulated architecture, when being emulated. should we take the opportunity to make this clear?

Reply:
That is a great point. I will enrich the remarks section to mention the emulation angle. I need to step away for a bit, but will update shortly.

Reply:
I found time :-)

Reply:
@danmoseley Please take a look at the phrasing.